src/Device/Driver/AR62xx.cpp

```diff
@@ -347,15 +322,19 @@ unsigned int AR62xxDevice::GenerateSetStationCommand(uint8_t *command, unsigned
 
   /** setting frequencies */
   command[command_length++] = 22;
-  command[command_length++] = GetNthByte(active_frequency_idx, 1);
-  command[command_length++] = GetNthByte(active_frequency_idx, 0);
-  command[command_length++] = GetNthByte(passive_frequency_idx, 1);
-  command[command_length++] = GetNthByte(passive_frequency_idx, 0);
-
-  /** Creating the binary value */
-  uint16_t command_crc = CRCBitwise(command, command_length);
-  command[command_length++] = GetNthByte(command_crc, 1);
-  command[command_length++] = GetNthByte(command_crc, 0);
+
+  uint16_t tmp[] = {0x00, 0x00};
```

Comment:
What's the point of initializing this array?

Reply:
replaced